src/audio/audio.cpp

```diff
@@ -87,23 +87,23 @@ std::unique_ptr<Response> PipeAudio::Emit(const std::string &path,
 
 	std::string value;
 
-	if (path == "/control/state") {
+	if (path == "/player/state/current") {
 		auto state = this->sink->State();
-		auto playing = state == Audio::State::PLAYING;
-		value = playing ? "Playing" : "Stopped";
+		value = Audio::STATE_STRINGS[static_cast<int>(state)];
+	} else if (path == "/player/state/available") {
+		// TODO: Use STATE_STRINGS?
+		value = "playing,stopped,ejected,finished";
 	} else if (path == "/player/file") {
 		value = this->src->Path();
 	} else if (path == "/player/time/elapsed") {
 		std::uint64_t micros = this->Position();
-
-		// Always announce a unicast.
-		// Only announce broadcasts if CanAnnounceTime(...).
-		bool can = (!broadcast) || this->CanAnnounceTime(micros);
-		if (!can) return ret;
 		value = std::to_string(micros);
-	} else return ret;
+	} else if (path == "/player/time/total") {
+		std::uint64_t micros = this->TotalLength();
+		value = std::to_string(micros);
+	}
 
-	return Response::Res("Entry", path, value);
+	return std::make_pair("Entry", value);
```

Comment:
First value of the pair should be `integer` for `/player/time/total` and `/player/time/elapsed`, and `string` for everything else, see [the types RFC](https://github.com/UniversityRadioYork/bifrost-rfcs/blob/master/protocol/types.md) and [the draft player RFC](https://github.com/UniversityRadioYork/bifrost-rfcs/blob/mw-player/resource-hierarchy/player.md).